schema/ClaimsAttributeObject.md

```diff
@@ -0,0 +1,36 @@
+# Attribute Object
+
+## Summary
+An Attribute Object contains attribute information the TDF system uses to enforce access control. Attributes are used in both the [PolicyObject](PolicyObject.md) to define the attributes that a user "needs" to gain access in an ABAC sense, and in the [ClaimsObject](ClaimsObject.md) to assert the attributes that a user "has" to satisfy the ABAC needs.
```

Comment:
Thoughts on using the term "entity" as opposed to "user"?

Reply:
We gotta ditch `user` from our lexicon anyway so all for that.